test: team booking

shivamklr opened the pull request:
## What does this PR do?

Test team bookings.


<!-- Please provide a loom video for visual changes to speed up reviews
 Loom Video: https://www.loom.com/
-->
[Loom Video](https://www.loom.com/share/ef19ed0a4ad54388b5db765e2e5cbdb0)

## Type of change

<!-- Please delete bullets that are not relevant. -->

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Write details that help to start the tests -->

- Run `teams.e2e.ts`

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

shivamklr committed:
test: Team booking

shivamklr committed:
Merge branch 'main' into test/team-booking

shivamklr reviewed (commented):
Self Review

emrysal reviewed (dismissed):
I believe you also need to add the member as a host to the team event type for this to work?

shivamklr committed:
fix: add success page assertion

shivamklr commented:
> I believe you also need to add the member as a host to the team event type for this to work?
We are doing that in the user fixture already. Added a new assertion for success page, for clarity. Updated Loom video.

shivamklr committed:
Merge branch 'main' into test/team-booking

alannnc committed:
Merge branch 'main' into test/team-booking

alannnc commented on apps/web/playwright/teams.e2e.ts:
```diff
@@ -62,4 +64,43 @@ test.describe("Teams", () => {
+        role: MembershipRole.MEMBER,
+        accepted: true,
+      },
+    });
```
Whats the teamMate2 role in this test? Should we edit event type to be collective event so the success booking have 2 team attendees and 1 guest?

shivamklr replied:
I think that is a valid point. We should consider a valid schedulingType.

alannnc reviewed (dismissed):
Left a comment about teamMember2 role in the test.

shivamklr committed:
Merge branch 'main' into test/team-booking

shivamklr committed:
fix: test Collective EventType

shivamklr committed:
fix: add robust assertions

shivamklr committed:
Merge branch 'main' into test/team-booking

shivamklr committed:
Merge branch 'main' into test/team-booking
